omnibus/config/projects/agent.rb

```diff
@@ -181,10 +181,19 @@
       "#{Omnibus::Config.source_dir()}\\datadog-agent\\src\\github.com\\DataDog\\datadog-agent\\bin\\agent\\trace-agent.exe",
       "#{Omnibus::Config.source_dir()}\\datadog-agent\\src\\github.com\\DataDog\\datadog-agent\\bin\\agent\\agent.exe",
       "#{Omnibus::Config.source_dir()}\\datadog-agent\\src\\github.com\\DataDog\\datadog-agent\\bin\\agent\\libdatadog-agent-three.dll",
-      "#{install_dir}\\bin\\agent\\ddtray.exe"
+      "#{install_dir}\\bin\\agent\\ddtray.exe",
+      "#{windows_safe_path(python_3_embedded)}\\python.exe",
```

Comment:
Reading the job logs, this line (and the next ones) are failing because the `python_3_embedded` variable isn't defined.
```
C:/dev/go/src/github.com/DataDog/datadog-agent/omnibus/config/projects/agent.rb:185:in `block in evaluate': undefined local variable or method `python_3_embedded' for #<Omnibus::Packager::MSI (Cleanroom)>
```

The reason is that here, you're in the packager block, and `python_3_embedded` is only defined for the `Project` class, not the `Packager::MSI` class.

I'm going to check the omnibus code to see what we could do here.